src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -889,6 +891,12 @@ MulticopterAttitudeControl::task_main()
 			vehicle_status_poll();
 			vehicle_motor_limits_poll();
 
+			/* Simple handling of failsafe: kill motors if failsafe is on */
+			if (_termination_enabled && _v_control_mode.flag_control_termination_enabled) {
+				_armed.lockdown = true;
```

Comment:
This won't have any effect. The `_armed` topic is subscribed here, not published.